consumers/google.go

```diff
@@ -65,10 +56,21 @@ func NewGoogleDNS() (Consumer, error) {
 		return nil, fmt.Errorf("Error creating DNS service: %v", err)
 	}
 
+	resp, err := client.ManagedZones.List(params.project).Do()
+	if err != nil {
+		return nil, fmt.Errorf("Error getting managed zones in project %s: %v", params.project, err)
+	}
+
+	zones := make(map[string]*dns.ManagedZone)
```

Comment:
this change btw should also help with the comment above about having a single 
`func (d *googleDNSConsumer) hostedZoneFor(name string) string {` for consumers